lib/extensions/polyglot_with_database/coursemology/polyglot/language/python/python2_point7.rb

```diff
@@ -0,0 +1,7 @@
+module Extensions::PolyglotWithDatabase::Coursemology::Polyglot::Language::Python::Python2Point7
```

Comment:
I don't want to introduce database into the gem because that would introduce a dependency of ActiveRecord into the gem, which the evaluator will also pull in...

Probably the language declarations will be in the gem, the web app just handles database and nothing else?